test/app/views/integration_examples/products.html.erb

```diff
@@ -1,4 +1,4 @@
 <h1>Products for sale</h1>
 <ul>
-  <%= render(ProductComponent.with_collection(@products, notice: "Today only")) %>
+  <%= render(ProductComponent.with_collection(@products, notice: "Today only", count: @products.count)) %>
```

Comment:
I like this. If you want the count you can do it yourself. We have to help to make the counter line up.

Reply:
Is this API present in Rails?